packages/wokwi-client-js/package.json

```diff
@@ -24,7 +24,9 @@
     }
   },
   "files": [
-    "dist"
+    "dist",
+    "../README.md",
```

Comment:
Same issue as above with relative paths. I suggest looking how other monorepo packages solve this.